content/en/docs/concepts/scheduling-eviction/outofresource-handling.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Out of resource Eviction handling 
+content_template: templates/concept
```

Comment:
Hello @santadasu . 
What do you think about adding a paragraph or section to the Eviction policy page instead
of creating a new concept page? There is already a link to the "out-of-resource" handling page.
See `https://kubernetes.io/docs/concepts/scheduling-eviction/eviction-policy/`.

Reply:
I thought that this needed a more prominent section. Hence did it this way.

As the link is already present, do you want me to just add an extra paragraph before the link?

Reply:
I don't think this adds any new content. The content is the same as the task page first paragraph and the first sentence in the second section. This is the task page content:

```
The kubelet needs to preserve node stability when available compute resources are low. This is especially important when dealing with incompressible compute resources, such as memory or disk space. If such resources are exhausted, nodes become unstable.

The kubelet supports eviction decisions based on the signals described in the following table. 
```

And this is the content you propose:
```
The `kubelet` needs to preserve node stability when available compute resources
are low. This is especially important when dealing with incompressible
compute resources, such as memory or disk space. If such resources are exhausted,
nodes become unstable.

The `kubelet` supports eviction decisions based on available memory and filesystem on both cluster nodes and containers. 
```

Reply:
@kbhawkey  The issue description says that it will be good to cut and paste the content in the target page. As the information is belongs to Concepts vs Tasks. Hence I used exact content.  

From what you say is there is no need to fix anything at all?

Reply:
Hello @santadasu .
First, would you verify that you have signed the CLA and that your email address associated with this PR matches the email address you used to sign the CLA?

Are you referring to this issue:
https://github.com/kubernetes/website/issues/20649
If so, you can assign yourself to the issue and ask any questions in the issue.

I would prefer that the content taken from the task page is added to an existing page, not duplicated with the content in the task page.

/hold cancel